### ISSUE TITLE
Issue: Schema version mismatch when deploying

### ISSUE THREAD

**Issue by nhu-mai-101** [maoosi/prisma-appsync#155]
Started running into this issue when attempting to deploy to AWS. Never had this issue before, but seemed to coincide with the aws-cdk/aws-cdk-lib [2.87.0] release 19 days ago.

```
This CDK CLI is not compatible with the CDK library used by your application. Please upgrade the CLI to the latest version.
(Cloud assembly schema version mismatch: Maximum schema version supported is 31.0.0, but found 32.0.0)
```

These are my AWS CDK CLI and AWS CLI versions:
cdk --version
2.88.0 (build 5d497f9)

aws --version
aws-cli/2.7.21 Python/3.9.11 Darwin/22.4.0 exe/x86_64 prompt/off

**Comment by maoosi** [comment]
Hey @nhu-mai-101. Here is a first suggestion:

1. Make sure to upgrade your AWS CDK CLI to the latest version. If using npm: `npm install -g aws-cdk@latest`. If using yarn: `yarn global upgrade aws-cdk`.
2. Confirm that you are using the same CDK boilerplate version provided from https://github.com/maoosi/prisma-appsync/tree/main/packages/boilerplate/cdk (was last updated in December).
3. Try to deploy again.

If it still doesn't work, then let me know and I'll try replicate the issue on my end.

**Comment by maoosi** [comment]
@nhu-mai-101 I have been unable to reproduce the issue you encountered.

However, I've released a new version, [1.0.0-rc.7](https://github.com/maoosi/prisma-appsync/releases/tag/1.0.0-rc.7). In this release, I've updated the CDK Boilerplate to use the most recent versions of all the available CDK packages.

This update should, in theory, resolve the issue you're experiencing. See the full list of changes applied: https://github.com/maoosi/prisma-appsync/commit/af5b3353debb2326b2d03ed144fb901d7b0c8e57
